README.md

```diff
@@ -1,2 +1,166 @@
-# Math-Magician
-Creating a calculator using react as front end and add it to a page to use it. This webpage contain three slides .
+# Math Magicians
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+Creating a calculator using react as front end and add it to a page to use it. This webpage contain three slides . 
+
+## 🛠 Built With <a name="built-with"></a>
+
+- React
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+    <li>HTML and CSS</li>
+    <li>Javascript & DOM</li>
+    
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Add calculator**
+- **Create 3 slide page**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo(still in development) <a name="live-demo"></a>
+
+
+- [live project](#) - upcoming 
+
+## 🚀 Live Video(still in development) <a name="live-video"></a>
```

Comment:
- [ OPTIONAL ] Since you don't have a live demo and live video, just add it still in development word without adding a link to it. It looks more professional